### ISSUE TITLE
Add value property

### ISSUE THREAD

**Issue by matteo-pennisi** [d-a-n/react-native-modal-picker#64]
By default data need to be formatted in this way:
```const data = [
            { key: index++,  label: 'Fruits' },
            { key: index++, label: 'Red Apples' },
            { key: index++, label: 'Cherries' },
            { key: index++, label: 'Cranberries' },
```
I really need a value property like [React Native Picker](https://facebook.github.io/react-native/docs/picker.html). For example input data will be:
```const data = [
            { key: index++,  label: 'Fruits', value: 'fruit1' },
            { key: index++, label: 'Red Apples', value: 'fruit5' },
            { key: index++, label: 'Cherries',value: 'fruit3'},
            { key: index++, label: 'Cranberries',value: 'fruit11' },
```
How I can do this?

**Comment by peacechen** [comment]
See https://github.com/peacechen/react-native-modal-selector/issues/2

**Comment by matteo-pennisi** [comment]
Closed in [peacechen#2](https://github.com/peacechen/react-native-modal-selector/issues/2)
